conans/cli/commands/test.py

```diff
@@ -0,0 +1,107 @@
+    out.highlight("\n-------- Input profiles ----------")
+    out.info("Profile host:")
+    out.info(profile_host.dumps())
+    out.info("Profile build:")
+    out.info(profile_build.dumps())
+
+    root_node = conan_api.graph.load_root_test_conanfile(path, ref,
+                                                         profile_host, profile_build,
+                                                         require_overrides=args.require_override,
+                                                         remote=remote,
+                                                         update=args.update,
+                                                         lockfile=lockfile)
+
+    out.highlight("-------- Computing dependency graph ----------")
+    check_updates = args.check_updates if "check_updates" in args else False
+    deps_graph = conan_api.graph.load_graph(root_node, profile_host=profile_host,
+                                            profile_build=profile_build,
+                                            lockfile=lockfile,
+                                            remote=remote,
+                                            update=args.update,
+                                            check_update=check_updates)
+    print_graph_basic(deps_graph)
+    out.highlight("\n-------- Computing necessary packages ----------")
+    conan_api.graph.analyze_binaries(deps_graph, remote=remote, update=args.update)
+    print_graph_packages(deps_graph)
+
+    out.highlight("\n-------- Installing packages ----------")
+    conan_api.install.install_binaries(deps_graph=deps_graph,
+                                       remote=remote, update=args.update)
+
+    if args.lockfile_out:
+        lockfile_out = make_abs_path(args.lockfile_out, cwd)
+        out.info(f"Saving lockfile: {lockfile_out}")
+        lockfile.save(lockfile_out)
+
+    out.highlight("\n-------- Testing the package ----------")
+
+    conanfile_folder = os.path.dirname(path)
+    conan_api.install.install_consumer(deps_graph=deps_graph, base_folder=cwd,
+                                       reference=ref, create_reference=True,
+                                       install_folder=conanfile_folder,
+                                       conanfile_folder=conanfile_folder)
+    conanfile = deps_graph.root.conanfile
+
+    if hasattr(conanfile, "layout"):
+        conanfile.folders.set_base_build(conanfile_folder)
+        conanfile.folders.set_base_source(conanfile_folder)
+        conanfile.folders.set_base_package(conanfile_folder)
+        conanfile.folders.set_base_generators(conanfile_folder)
+        conanfile.folders.set_base_install(conanfile_folder)
+    else:
+        conanfile.folders.set_base_build(conanfile_folder)
+        conanfile.folders.set_base_source(conanfile_folder)
+        conanfile.folders.set_base_package(conanfile_folder)
+        conanfile.folders.set_base_generators(conanfile_folder)
+        conanfile.folders.set_base_install(conanfile_folder)
```

Comment:
Empty diff?

Reply:
This is because the command has removed at the moment the ``--test-build-folder`` argument, indeed. The idea is that maybe the standard "--build-folder`` would apply for this case if necessary. (The ``--test-build-folder`` would only be needed for ``conan create``, but we could use ``--build-folder`` as well?) 

I will remove the non-sense if(), and we will see how the general ``--build-folder`` moves forward, but better in a new thread?